www/src/examples/Button/ToggleButtonGroupControlled.js

```diff
@@ -1,31 +1,15 @@
-class ToggleButtonGroupControlled extends React.Component {
-  constructor(props, context) {
-    super(props, context);
-
-    this.handleChange = this.handleChange.bind(this);
-
-    this.state = {
-      value: [1, 3],
-    };
-  }
-
-  handleChange(value, event) {
-    this.setState({ value });
-  }
-
-  render() {
-    return (
-      <ToggleButtonGroup
-        type="checkbox"
-        value={this.state.value}
-        onChange={this.handleChange}
-      >
-        <ToggleButton value={1}>Option 1</ToggleButton>
-        <ToggleButton value={2}>Option 2</ToggleButton>
-        <ToggleButton value={3}>Option 3</ToggleButton>
-      </ToggleButtonGroup>
-    );
-  }
+function ToggleButtonGroupControlled() {
+  const [value, setValue] = React.useState([1, 3]);
+
+  const handleChange = val => setValue(val);
```

Comment:
Can we just do `onChange={setValue}` below?

Reply:
hmm, I don't think that will work since `ToggleButtonGroup` passes 2 arguments to the `onChange` prop when it gets fired.
https://github.com/react-bootstrap/react-bootstrap/blob/391af529b750a106264c37871bdff6d7df00bc89/src/ToggleButtonGroup.js#L58
Technically this should be written as
```js
const handleChange = (val, event) => setValue(val);
```
or more simply
```js
const handleChange = (val, ) => setValue(val);
```

Reply:
ah okay